remediation/self-node-remediation/how-it-works.md

```diff
@@ -36,3 +36,30 @@ If there's an api-server failure, which means that all nodes can't access the ap
 It would create an unnecessary reboot storm in the cluster.
 
 ![self-node-remediation-api-server-failure.png](../../images/self-node-remediation-api-server-failure.png)
+
+## Control plane fencing
+Control plane fencing can be divided into two primary scenarios.
```

Comment:
> Control plane fencing can be divided into two primary scenarios.

It is straightforward, but might be nicer to deliberately mention the scenarios, as it "preparing" the reader of what comes next.
`Control plane fencing can be divided into two primary scenarios` -> `Control plane fencing can be divided into two primary scenarios, when there is an API Server Connectivity, and when there is no such connection`